error.go

```diff
@@ -129,7 +129,7 @@ type ActiveEndpointsError struct {
 }
 
 func (aee *ActiveEndpointsError) Error() string {
-	return fmt.Sprintf("network %s has active endpoints", aee.name)
+	return fmt.Sprintf("network %s id %s has active endpoints", aee.name, aee.id[0:12])
```

Comment:
yes yes, agreed that is a useful information, I was more concerned by the risk of the slice out of bound, but I saw now that also in other part of the code there is the hard assumption on the size like id[:12]
Can you in case uniform it using the same notation without the 0 (aee.id[:12])?